docs/governance/steering/charter.md

```diff
@@ -0,0 +1,177 @@
+* Appoint leadership for Special Interest Groups (SIGs).
+* Define and evolve project and group<sup>[3](#footnote3)</sup> governance structures and policies<sup>[4](#footnote4)</sup>.
+* Act as a final non-technical escalation point for any CCC community group or technical effort.
+* Define and enforce requirements for community groups<sup>[3](#footnote3)</sup> to ensure ongoing positive standing, or decomission any groups not in good standing.
+* Decide who is a member of standing on the CCC project, and what privileges that entails.
+* Control and delegate access to and establish processes regarding project resources<sup>[5](#footnote5)</sup>.
+* Coordinate with FINOS regarding usage of the CCC brand and deciding which things can be called "Common Cloud Controls," as well as how that mark can be used in relation to other efforts or vendors.
+* Request funds and other support from FINOS (e.g. marketing, press, etc).
+* Interface with the FINOS Staff point of contact as needed for guidance and accountability.
+
+### FINOS Point of Contact
+
+In addition to general foundation support given to projects, the assigned point of contact for the steering committee to coordinate with FINOS Staff is:
+
+Rob Moffat — [@robmoffat](https://github.com/robmoffat)
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members and shall always be considered this size regardless of temporary vacancies, for the purpose of voting majority and supermajority considerations.
+
+**FSI Seats —** Four (4) seats on the committee are reserved for FINOS members from financial services institutions. These seats may not be occupied by more than one individual from the same institution<sup>[6](#footnote6)</sup>. In the event that a committee member occupying an FSI seat undergoes a change of employment, they may retain their seat while in noncompliance for 30 calendar days before it is automatically considered vacant.
+
+**Community Seats —** Three (3) seats on the committee are not reserved, and may be filled by any community member from any organization.
+
+### Elections
+
+Every year, the Steering Committee holds a general election for open seats.
+
+Our [election policy document][/steering-elections.md] covers the details for how this works.
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected committee member, the next most preferred candidate from the previous election will be offered the seat.
+
+A maximum of one (1) committee member may be selected this way between elections.
+
+In case this fails to fill the seat, a special election for that position will be held as soon as possible.
+
+[Eligible voters][voter-eligibility] from the most recent election will vote in the special election i.e., eligibility will not be redetermined at the time of the special election.
```

Comment:
Broken link - I get 404 when I click on it

![image](https://github.com/finos/common-cloud-controls/assets/20100558/665ce3da-7e07-4b41-8bd9-809203296252)

Reply:
I think this is to do with the GitHub code review tool.  If you click the ellipsis top right and "view file" then try the links, they work.  Or, at least the ones I tested did.